### DISCUSSION TITLE
Logout from all devices or session timeout

### DISCUSSION THREAD

**Discussion by 1RandomDev** [coder/code-server#4538] [Q&A] [answered]
I'm using my codeserver instance in IT class at school and constantly forget to log out.

Is there an option where I can set an automatic timeout after I logged in or maybe when closing the browser. Or is there at least a possibility to log out of all devices if I forget it?

**Comment by lg-nineoneninethreenine** [comment]
I think there is kind of a logout timer or something but only @jsjoeio or @code-asher might know about this in depth.

**Comment by jsjoeio** [comment, marked as answer]
> Is there an option where I can set an automatic timeout after I logged in or maybe when closing the browser.

Not currently possible, but feel free to open an issue and we can track it as a feature request!
